Cargo.lock

```diff
@@ -934,11 +939,87 @@ dependencies = [
  "winapi 0.3.4 (registry+https://github.com/rust-lang/crates.io-index)",
 ]
 
+[[package]]
+name = "rand"
+version = "0.6.3"
```

Comment:
Rand 0.4, 0.5, and 0.6... 😭

Reply:
Oh boy, I hadn't noticed. It would be nice if we could do something about that...

Reply:
Looks like the `rand` 0.4 dependency is pulled in by `tempdir`, which is a dependency of `prost-build`. The `tempdir` crate is deprecated in favour of `tempfile`, but we use it in our tests as well (to pull in fewer dependencies).

I'll look at a patch to `prost-build` replacing `tempdir` with `tempfile`, and hopefully we can get rid of `rand` 0.4

Reply:
@seanmonstar if we merge both this branch and #170, we will depend exclusively on the latest `rand` (0.6.3). 

Sorry it took a bit, the dependency update was a surprisingly large amount of work.